packages/web-react/src/components/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+type VisuallyHiddenProps = {
+  children: string;
+  elementType?: 'span' | 'div' | 'legend';
+};
+
+const defaultProps = {
+  elementType: 'span',
+};
+
+const VisuallyHidden = (props: VisuallyHiddenProps) => {
+  const { children, elementType: ElementTag = 'span' } = props;
+
+  return <ElementTag className="accessibility-hidden">{children}</ElementTag>;
```

Comment:
Beware, this class does not support prefixing and it should, I think.

Reply:
does utility classes have prefixes? i think it should be without prefix

Reply:
They do. We suggest prefixing everything except is-* or has-* classes. See https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web#prefixing-css-class-names

Reply:
@pavelklibani So the logic with prefixing and the test should be added. Please do so :-)

Reply:
prefixing added